File: HerenciaCSharp/HerenciaCSharp/Persona.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HerenciaCSharp
{
    public class Persona
    {
        private String Nombre;
        private String Telefono;

        public void setNombre(String val)
        {
            Nombre = val;
        }

        public String getNombre()
        {
            return Nombre;
        }

        public void setTelefono(String val)
        {
            Telefono = val;
        }

        public String getTelefono()
        {
            return Telefono;
        }
    }
}

File: HerenciaVisual/HerenciaVisual/Persona.vb
Public Class Persona
    Private nombre As String
    Private telefono As String

    Sub New()
    End Sub

    Sub New(ByVal nombre1 As String, ByVal telefono1 As String)
        Me.nombre = nombre1
        Me.telefono = telefono1
    End Sub

    Sub New(ByVal nombre1 As String)
        Me.nombre = nombre1
    End Sub

    Public Function GetNombre() As String
        Return Me.nombre
    End Function

    Public Function GetTelefono() As String
        Return Me.telefono
    End Function

    Public Sub SetNombre(ByVal nombre1 As String)
        nombre = nombre1
    End Sub

    Public Sub SetTelefono(ByVal telefono1 As String)
        telefono = telefono1
    End Sub

    Overridable Sub Imprimir()
        MessageBox.Show(Me.nombre)
        MessageBox.Show(Me.telefono)
    End Sub

End Class
